driver.go

```diff
@@ -223,6 +227,13 @@ func (l *lvmDriver) Remove(req *volume.RemoveRequest) error {
 	if err != nil {
 		return err
 	}
+	vol, exists := l.volumes[req.Name]
+	if !exists {
+		return fmt.Errorf("Error removing volume, missing description in lvmConfigVolumes.json")
+	}
```

Comment:
This error is returned back to docker client. docker client doesn't know about plugin config files.
So something like `missing description in lvmConfigVolumes.json` in the error message is irrelevant to docker client.

Would be better to use error message/format same as L285:
`return &volume.MountResponse{}, fmt.Errorf("Unknown volume %s", req.Name)`